### PR TITLE
[ASK DAN BEFORE MERGING, expected mid-Jan] Maker deprecation warnings

### PR DESCRIPTION
**Pull request by bencodeorg**
Adds deprecation warnings to Maker setup and homepages.

**/maker/setup** (first notification visible to Maker app users, second notification visible to old Chrome versions. No one should see both notifications)

image

**/maker/home** (notification visible to everyone)

image

## Links

- jira ticket: [SL-404](https://codedotorg.atlassian.net/browse/SL-404)

## Testing story

Tested manually that the Chrome version detection worked for me locally. I did not test the Maker app (I don't have it set up locally, and the code I'm adding seems pretty straightforward).

## Deployment strategy

This should be merged as part of Maker deprecation messaging in January -- check in with Dan for details.

### CODE REVIEW

File: apps/src/lib/kits/maker/ui/SetupGuide.jsx [hunk 1]
```diff
@@ -49,11 +53,34 @@ export default class SetupGuide extends React.Component {
         };
     }
   };
+
   render() {
     // Experiment 'microbit', displays Circuit Playground and Micro:Bit descriptions.
-    let isMicrobit = experiments.isEnabled('microbit');
+    const isMicrobit = experiments.isEnabled('microbit');
```
Comment (bencodeorg) [inline apps/src/lib/kits/maker/ui/SetupGuide.jsx:61]: Unrelated cleanup.

File: apps/src/lib/kits/maker/ui/SetupGuide.jsx [hunk 2]
```diff
@@ -49,11 +53,34 @@ export default class SetupGuide extends React.Component {
...
+    const chromeVersion = getChromeVersion();
+
     return (
       <div>
+        {isCodeOrgBrowser() && (
+          <Notification
```
Comment (molly-moen) [inline apps/src/lib/kits/maker/ui/SetupGuide.jsx:67]: should we have two banners here? Is it possible to combine these into one message?

Comment (bencodeorg) [reply apps/src/lib/kits/maker/ui/SetupGuide.jsx:67]: I don't think anyone would see both...the first is for the Maker app, and the second for old chrome versions? Unless I'm misunderstanding something

Comment (molly-moen) [reply apps/src/lib/kits/maker/ui/SetupGuide.jsx:67]: ah ok I was confused because your screenshot had both

**Review by molly-moen — APPROVED** [review]

File: apps/src/lib/kits/maker/ui/SetupGuide.jsx (hunk 2 above)
Comment (bencodeorg) [reply apps/src/lib/kits/maker/ui/SetupGuide.jsx:67]: ah apologies, was trying to minimize screenshot screen real estate -- should have split those out!
